### ISSUE TITLE
Can it be applied to the waymo dataset?

### ISSUE THREAD

**Issue by Wata-Brian** [tianweiy/MVP#29]
I'm using centerpoint using waymo dataset.
I haven't used the nuscence dataset
Is it difficult to apply MVP to the waymo dataset?
Is it possible to just use it as a different parameter value?

**Comment by tianweiy** [comment]
I briefly tried this two years ago but didn't manage to get good results back then. I guess it needs a bit more bells and whistles. 

On the other hand, I do noticed a few papers on the similar direction e.g. https://github.com/hailanyi/VirConv (also check the papers that cite MVP).

**Comment by yangyt37** [comment]
The reason why I don't think it can achieve good results in waymo is that waymo uses 64 line lidar, which already has sufficient front attractions. Continuing to add virtual points may not be enough for 32 line nuscenes. Similarly in kitti datasets, if you have any understanding, please correct it!

**Comment by tianweiy** [comment]
yeah, I think this is the main reason. Though I think there are a few more consideration , for instance https://github.com/LittlePey/SFD and https://github.com/hailanyi/VirConv also use virtual points and work on kitti. from my understanding, the main difference is the place we do the fusion (sfd does it during roi refinement stage) and the modules we use to fuse virtual and real points (sfd / virconv gets more complex and better one)
